components/molecule/quickAction/src/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const CLASS = 'sui-MoleculeQuickAction'
+
+const SIZES = {
+  LARGE: 'large'
+}
+
+const getClassName = variant => `${CLASS}--${variant}`
+
+const MoleculeQuickAction = ({
+  size,
```

Comment:
What about a default value? `S`, `M` or whatever? This means to update the `Sizes`.